notification/notification-sdk/src/main/java/org/sunbird/notification/email/Email.java

```diff
@@ -111,6 +115,14 @@ private void initProps() {
      */
     props.put("mail.smtp.auth", "true");
     props.put("mail.smtp.port", port);
+    logger.info("The value of TLS Enabled is : " + isTlsEnabled);
+    props.put("mail.smtp.starttls.enable", "true");
+    props.put("mail.smtp.ssl.protocols", "TLSv1.2");
+    if ("true".equalsIgnoreCase(isTlsEnabled)) {
```

Comment:
use constant instead of "true"

Reply:
The above two lines of TLS getting enabled unconditionally is removed and is set inside if(if isTLS enabled = ture)